advisor-ui/src/components/OpportunityModal/OpportunityModal.jsx

```diff
@@ -2,16 +2,19 @@ import React from "react";
 import "./OpportunityModal.css";
 
 export default function OpportunityModal({ eventItem, handleCloseModal }) {
+  // YYYY-MM-DD = 10 chars
+  const standardDateLength = 10;
+
   return (
     <div className="oppmodal">
       <div className="content">
         <button onClick={handleCloseModal}>Back</button>
         <h3>{eventItem.title}</h3>
         <div className="location">
-          <p>{eventItem.location}</p>
+          <p>{`${eventItem.city}, ${eventItem.state}`}</p>
         </div>
         <div className="date">
-          <p>{eventItem.date}</p>
+          <p>{eventItem.date.slice(0, standardDateLength)}</p>
```

Comment:
It's more common to pass around dates as unix timestamps, IDR off hand if you're converting it to some format on the backend or this is just how it's returned. There's a JS method `new Date(timestamp).toDateLocale()` which is nice because it'll handle formatting (mm/dd/yyyy vs dd/mm/yyyy) for you based on what the user's system/browser is using.

Reply:
Seems like sequalize doesn't really give a great/easy way of using timestamp as ms rather than storing a user-specific time. 

I'd still try to see if you can get it to work using the built in JS `Date` class rather than slicing it like this though. Just feels a bit... fishy.